### PR TITLE
ui: Support to view template download progress across all stores

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description

This PR provides a means of viewing the download progress across all the stores in a zone

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

### How Has This Been Tested?
image

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]
Tested! LGTM!
image

**Review by shwstppr — APPROVED** [review]
Functionality works.
Though I'm not sure about the use of the term `Datastore`. A little bit confusing as when using VMware as hypervisor all storage show as Datastore there and these are secondary storage only.

**Comment by Pearl1594** [comment]
> Functionality works.
> Though I'm not sure about the use of the term `Datastore`. A little bit confusing as when using VMware as hypervisor all storage show as Datastore there and these are secondary storage only.

@shwstppr I've updated the column name to secondary storage 
image

**Review by sureshanaparti — APPROVED** [review]
clgtm

File: ui/public/locales/en.json [hunk 1]
```diff
@@ -654,6 +654,7 @@
 "label.dark.mode": "Dark mode",
 "label.data.disk": "Data Disk",
 "label.data.disk.offering": "Data Disk Offering",
+"label.datastore": "Datastore",
```
Comment (shwstppr) [inline ui/public/locales/en.json:657]: Can be removed if not used

**Review by shwstppr — APPROVED** [review]
LGTM w/ one minor nit

**Review by shwstppr — APPROVED** [review]

**Comment by nvazquez** [comment]
@blueorangutan ui

**Review by nvazquez — APPROVED** [review]

**Pull request merged**